package.json

```diff
@@ -75,6 +76,7 @@
     "gl": "^4.0.4",
     "glob": "^7.1.2",
     "jest": "^21.2.1",
+    "jest-cli": "^22.0.4",
```

Comment:
why is this needed?

Reply:
I was getting test errors saying it was not available, I added that and it worked -- but I can remove it and see if my tests fail.

Reply:
I can't run npm run test:js without it

Reply:
How have we been running tests for the past few months? It sure has been working for me. Did it get removed perhaps?

Reply:
well... i was cheating, installing it locally on my branch everytime, and this time I just got tired of doing it..

Reply:
I guess it's because I always run tests like `../node_modules/jest/bin/jest` instead of using `npm run test:js` :P  ? Well our npm scripts ought to work so :+1:

Reply:
Though how does it work in Circle CI? Puzzling.